packages/fxa-auth-server/lib/routes/linked-accounts.ts

```diff
@@ -19,20 +19,29 @@ import isA from 'joi';
 import DESCRIPTION from '../../docs/swagger/shared/descriptions';
 import error from '../error';
 import { schema as METRICS_CONTEXT_SCHEMA } from '../metrics/context';
+import {
+  getGooglePublicKey,
+  isValidClientId,
+  validateSecurityToken,
+  eventHandlers,
+} from './utils/third-party-events';
 
 const HEX_STRING = validators.HEX_STRING;
 
-const appleAud = 'https://appleid.apple.com';
+const APPLE_AUD = 'https://appleid.apple.com';
 
 export class LinkedAccountHandler {
   private googleAuthClient?: OAuth2Client;
   private otpUtils: any;
+  private goooglePublicKey: any;
+
   constructor(
     private log: AuthLogger,
     private db: any,
     private config: ConfigType,
     private mailer: any,
-    private profile: ProfileClient
+    private profile: ProfileClient,
+    private statsd: any
```

Comment:
nit: maybe give it a type so that it cannot be `undefined`?  it can just be what it's used `{increment: () => void}`.